test/src/test/java/hudson/cli/AbstractBuildRangeCommandTest.java

```diff
@@ -134,58 +134,58 @@
                 .authorizedTo(Jenkins.READ, Job.READ)
                 .invokeWithArgs(PROJECT_NAME, "1");
         assertThat(result, succeeded());
-        assertThat(result.stdout(), containsString("Builds: 1\n"));
+        assertThat(result.stdout(), containsString("Builds: 1"+System.lineSeparator()));
 
         // First with plus symbol '+'
         result = command
                 .authorizedTo(Jenkins.READ, Job.READ)
                 .invokeWithArgs(PROJECT_NAME, "+1");
         assertThat(result, succeeded());
-        assertThat(result.stdout(), containsString("Builds: 1\n"));
+        assertThat(result.stdout(), containsString("Builds: 1"+System.lineSeparator()));
 
         // In the middle
         result = command
                 .authorizedTo(Jenkins.READ, Job.READ)
                 .invokeWithArgs(PROJECT_NAME, "10");
         assertThat(result, succeeded());
-        assertThat(result.stdout(), containsString("Builds: 10\n"));
+        assertThat(result.stdout(), containsString("Builds: 10"+System.lineSeparator()));
 
         // In the middle with plus symbol '+'
         result = command
                 .authorizedTo(Jenkins.READ, Job.READ)
                 .invokeWithArgs(PROJECT_NAME, "+10");
         assertThat(result, succeeded());
-        assertThat(result.stdout(), containsString("Builds: 10\n"));
+        assertThat(result.stdout(), containsString("Builds: 10"+System.lineSeparator()));
```

Comment:
containsString doesn;t take a string format according to the docs?  Are you proposing to use `contasintString(String.format("Builds: 10%n"))`  - in which case this looks ugly - so a matcher `containsFormattedString(String, Object...)` would be cleaner, or even a `containsLine(String)` that adds the line end...